configs/config.novacustom_ns5x_adl

```diff
@@ -14,7 +14,7 @@ CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/novacustom/bootsplash.bmp"
 CONFIG_TPM_MEASURED_BOOT=y
 CONFIG_POWER_STATE_OFF_AFTER_FAILURE=y
 CONFIG_ENABLE_EARLY_DMA_PROTECTION=y
-CONFIG_EC_SYSTEM76_EC_UPDATE=y
+CONFIG_EC_SYSTEM76_EC_UPDATE=n
```

Comment:
Please revert, let's keep it enabled in CI builds

Reply:
Reverted [here](https://github.com/Dasharo/coreboot/pull/513/files#diff-b6dec654230c58a4e16eede8e4703521eca64baaa11d381c93a4eb28bfd2c3feR17)